drivers/storage/portworx/manifest/manifest.go

```diff
@@ -85,6 +85,8 @@ type Release struct {
 	KubeScheduler              string `yaml:"kubeScheduler,omitempty"`
 	KubeControllerManager      string `yaml:"kubeControllerManager,omitempty"`
 	Pause                      string `yaml:"pause,omitempty"`
+	DynamicPlugin              string `yaml:"dynamicPlugin,omitempty"`
+	Nginxc                     string `yaml:"nginxc,omitempty"`
```

Comment:
Why do we have the `c` as the suffix? Can it be just `nginx` or `dynamicPluginProxy`?

Reply:
changed to DynamicPluginProxy name